spec/std/env_spec.cr

```diff
@@ -42,4 +42,36 @@ describe "ENV" do
     ENV["BAR"] = "SOMEVALUE_2"
     [1,2].each {|i| ENV.values.should contain("SOMEVALUE_#{i}") }
   end
+
+  describe "fetch" do
+    it "fetches with one argument" do
+      ENV["1"] = "2"
+      ENV.fetch("1").should eq("2")
+    end
+
+    it "fetches with default value" do
+      ENV["1"] = "2"
+      ENV.fetch("1", "3").should eq("2")
+      ENV.fetch("2", "3").should eq("3")
+    end
+
+    it "fetches with block" do
+      ENV["1"] = "2"
+      ENV.fetch("1") { |k| k + "block" }.should eq("2")
+      ENV.fetch("2") { |k| k + "block" }.should eq("2block")
+    end
+
+    it "coerces the value of the block to a string" do
```

Comment:
@jhass I was planning to drop this test rather than rewriting it because I'm guessing you don't test explicit type restrictions, but when I remove 66 and 67 I get a segmentation fault when running the spec.